<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XcityVideo</class>
 <widget class="QWidget" name="XcityVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="XcityJpDvdFinish">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="prefix">
    <string>End : </string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>57000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="XcityJpDvdStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="prefix">
    <string>Start : </string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="XcityJpDvdInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>440</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>xcity.jp DVD information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="XcityJpDvdId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>440</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>xcity.jp DVD identifier</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="XcityJpSkip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip existing records</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="XcityJpDvdGallery">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>440</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>xcity.jp DVD gallery</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="XcityNames">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">2</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="XcityJpCache">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Use cache</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>XcityJpSkip</sender>
   <signal>stateChanged(int)</signal>
   <receiver>XcityVideo</receiver>
   <slot>XcityValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XcityJpDvdStart</sender>
   <signal>editingFinished()</signal>
   <receiver>XcityVideo</receiver>
   <slot>XcityJpChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XcityJpDvdGallery</sender>
   <signal>editingFinished()</signal>
   <receiver>XcityVideo</receiver>
   <slot>XcityJpChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XcityJpDvdInfo</sender>
   <signal>editingFinished()</signal>
   <receiver>XcityVideo</receiver>
   <slot>XcityJpChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XcityJpDvdId</sender>
   <signal>editingFinished()</signal>
   <receiver>XcityVideo</receiver>
   <slot>XcityJpChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XcityJpDvdFinish</sender>
   <signal>editingFinished()</signal>
   <receiver>XcityVideo</receiver>
   <slot>XcityJpChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XcityNames</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>XcityVideo</receiver>
   <slot>NameClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XcityJpCache</sender>
   <signal>stateChanged(int)</signal>
   <receiver>XcityVideo</receiver>
   <slot>XcityJpChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
